Directory.Build.props

```diff
@@ -54,7 +54,7 @@
     <NetCoreAppToolCurrent>net$(NetCoreAppToolCurrentVersion)</NetCoreAppToolCurrent>
     <NetCoreAppCurrentToolTargetFrameworkMoniker>$(NetCoreAppCurrentIdentifier),Version=v$(NetCoreAppToolCurrentVersion)</NetCoreAppCurrentToolTargetFrameworkMoniker>
 
-    <NetCoreAppLatestStableVersion>5.0</NetCoreAppLatestStableVersion>
+    <NetCoreAppLatestStableVersion>6.0</NetCoreAppLatestStableVersion>
```

Comment:
I believe for this we need to take my PR which I'm working on at the moment: https://github.com/dotnet/runtime/pull/61437